xcconfig/xcconfig_test.go

```diff
@@ -27,3 +29,33 @@ func Test_WhenWritingXCConfigContent_ThenItShouldReturnFilePath(t *testing.T) {
 		assert.Equal(t, expectedPath, path)
 	}
 }
+
+func Test_XCConfigInput_NonExistentPathErrors(t *testing.T) {
```

Comment:
nit: I would follow the `// Given/When/Then` grouping convention in the new test cases too for consistency.

Reply:
Cool. Fixed it up.